Externals/FreeSurround/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+set(SRCS
+	ChannelMaps.cpp
+	KissFFT.cpp
+	KissFFTR.cpp
+	_KissFFTGuts.h
+	FreeSurroundDecoder.cpp
+	)
+
+add_library(FreeSurround STATIC ${SRCS})
```

Comment:
I did not change this yet as I was thinking how I should also configure Visual Studio to make it work on both. Will do later.